CONTRIBUTING.rst

```diff
@@ -61,4 +61,4 @@ Step-by-step release
 
 - Tag the merged commit in the format ``vMAJOR.MINOR.PATCH`` and push the tag to the primary repository.
 
-  - This will result in another build, which will generate the official release artifacts.
+  - This will result in another build, which will generate the official release artifacts.
```

Comment:
Is there a difference on this row? Spacing or something?

Reply:
ill revert